package.json

```diff
@@ -192,7 +192,7 @@
             "type": "string"
           },
           "default": [],
-          "description": "Optional array of strings to pass to the language server. Only works with Dafny versions above 3.10"
+          "description": "Optional array of strings to pass to the language server. Only works with Dafny versions above 3.10. Available options can be seen using 'dafny server --help'"
```

Comment:
That's a great idea. Because some users might not have Dafny in their code path, could you perhaps also provide a link to the document reference about these options?

Reply:
We don't have such a document reference yet. .NET as command specific documentation pages, like here: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-build, but we don't have that.

Reply:
Could you add the documentation of `dafny server --help` in the Dafny help, in another PR? You could use the same mechanism as "options.txt" to keep this help up to date. I think it would be really helpful, as we expect new users to not want to install Dafny manually to run this command, yet have access to the documentation of all the options they can now configure on VSCode thanks to your PRs.
Here is a good place to add this documentation. Perhaps it needs some refactoring.
https://dafny.org/latest/DafnyRef/DafnyRef#sec-dafny-language-server-vscode

Reply:
@davidcok is this something you could incorporate in https://github.com/dafny-lang/dafny/pull/3220 ?